SimonImanigirimpuhwe/my-brand-website
# #48 ft(blog): delete blog in DB

#### What does this PR do?

- Delete article in DB

#### Description of Task to be completed?

- Query all blogs in DB

- Select a blog you want to delete

- Delete article

#### How should this be manually tested?

- Clone the repo

- In the directory cd this branch ` ft-delete-existing-article-#48`

- In your code editor launch live server 

- If you have an admin account login and get redirected to the dashboard

- Click on `Posts` 

> You should see list of articles in DB with management options

> Then click on `Trash` button you should see the option to confirm or cancel

#### Any background context you want to provide?

-To be able to delete an article  you should have an admin account

#### What are the relevant Trello stories?

- [#48](https://trello.com/c/pZZ7lw8i/48-delete-existing-article)

#### Screenshots (if appropriate)

![delete](https://user-images.githubusercontent.com/54619678/95454182-632ddc80-096c-11eb-84b4-cda143b80a6a.png)


#### Questions:

- N/A
